src/main/riddl/Events/eventContext.riddl

```
@@ -0,0 +1,197 @@
+                }
+                on command AddLiveUpdate{???}
+            }
+        }
+        state InProgress is {
+            fields{
+                eventId: EventId,
+                eventInfo: EventInfo,
+                eventMetaInfo: EventMetaInfo
+            }
+            handler InProgressEventHandler is {
+                on command ScheduleEvent{ ??? // is this allowed or does an event have to be delayed first?
+                }
+                on command ChangeEventInfo{ ??? // can info be changed while event is in progress? Would this be more likely to confuse or help users?
+                }
+                on command CancelEvent{
+									then "morph Event entity to EventStatus.Cancelled"
+                  and "set entity lastModifiedBy to Event.cancellingMember"
+                  and "set entity lastModifiedOn to today()"
+                }
+                on command RescheduleEvent{ ??? // is this allowed or does the event have to be delayed first?
+                }
+                on command DelayEvent{ ??? // is this allowed or return error?
+                }
+                on command StartEvent{
+                  then error "Event is already started. Cannot start an event that is already in progress"
+                }
+                on command EndEvent{
+									then "morph Event entity to EventStatus.Past"
+                  and "set entity lastModifiedBy to Event.endingMember"
+                  and "set entity lastModifiedOn to today()"
+                }
+                on command AddLiveUpdate{???}
+            }
+            //score, period, etc...
+        }
+        state Delayed is {
+            fields {
+                eventId: EventId,
+                eventInfo: EventInfo,
+                reason: String,
+                eventMetaInfo: EventMetaInfo
+                //finalScore
+            }
+            handler DelayedEventHandler is {
+                on command ScheduleEvent{
```

I think we should throw an error if they try to schedule a Delayed event. We should direct them to reschedule if that is what they are trying to do.